Customizable recovery payload (#1283)
SumoLogic/sumologic-documentation: docs/manage/connections-integrations/webhook-connections/set-up-webhook-connections.md

    :::
1. (Optional) If the third-party system requires an **Authorization Header**, enter it here. For more information, see [Example Authorization Header](#example-authorization-header) below.
1. (Optional) **Custom Headers**, enter up to five comma separated key-value pairs.

> 1. For **Payload**, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. When using a generic webhook connection for notifications in a [Monitor](/docs/alerts/monitors) you can distinguish between an alert and a recovery by using the [alert variable](../../../alerts/alert-variables.md) `TriggerType`. An alert will have a value of either `Critical` , `Warning`, or `Missing ``Data`. Recovery will have either `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. The same payload for both alerts and recovery is used.

Reviewer: "When using a generic webhook connection for notifications in a [Monitor](/docs/alerts/monitors) you can distinguish between an alert and a recovery by using the [alert variable](../../../alerts/alert-variables.md)" 

This is no longer valid. We should remove this sentence. Since we now have a different payload for Generic Webhook too. That is how users can distinguish between Alert and Recovery

> 1. Under **Alert Payload**, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. When using a generic webhook connection for notifications in a [Monitor](/docs/alerts/monitors) you can distinguish between an alert and a recovery by using the [alert variable](../../../alerts/alert-variables.md) `TriggerType`. An alert will have a value of `Critical`, `Warning`, or `Missing Data`. Recovery will have `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. <br/><img src={useBaseUrl('img/connection-and-integration/alert-payload.png')} alt="recovery payload" />

Reviewer: "Alert Payload" 

Here we should describe in a line or two what is alert payload vs recovery payload. 

Alert Payload allows users to customize how the alert notification will look like. 

Recovery Payalod allows users to customize how the Recovery notification for an alert looks like.

> 1. Under **Alert Payload**, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. When using a generic webhook connection for notifications in a [Monitor](/docs/alerts/monitors) you can distinguish between an alert and a recovery by using the [alert variable](../../../alerts/alert-variables.md) `TriggerType`. An alert will have a value of `Critical`, `Warning`, or `Missing Data`. Recovery will have `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. <br/><img src={useBaseUrl('img/connection-and-integration/alert-payload.png')} alt="recovery payload" />

Author: ```suggestion
1. Under **Alert Payload**, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. An alert will have a value of `Critical`, `Warning`, or `Missing Data`. Recovery will have `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. <br/><img src={useBaseUrl('img/connection-and-integration/alert-payload.png')} alt="recovery payload" />
```

    :::note
    Variables are escaped according to the JSON standard, meaning that they can be used in application JSON. 
    :::
1. (Optional) Under **Recovery Payload**, you can customize your resolution notification payload JSON.<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload.png')} alt="recovery payload" /><br/>Example:<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload-example.png')} alt="recovery payload" />

> 1. Click **Save**.

Reviewer: There are also two button at the bottom of the page.

Test Alert Payload 
Test Alert + Recovery Payload 

We should mention what these buttons do. 

As the name suggests, these buttons allow user to test their alert and recovery payload after they have customized it. When a user clicks these buttons, Sumo Logic fires a sample webhook call to the given endpoint with the mentioned payload.

Author: Thank you! Will note this.

    :::
1. (Optional) If the third-party system requires an **Authorization Header**, enter it here. For more information, see [Example Authorization Header](#example-authorization-header) below.
1. (Optional) **Custom Headers**, enter up to five comma separated key-value pairs.

> 1. (Optional) Under **Alert Payload**, which allows you to customize how the alert notification will look, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. An alert will have a value of `Critical`, `Warning`, or `Missing Data`. Recovery will have `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. <br/><img src={useBaseUrl('img/connection-and-integration/alert-payload.png')} alt="recovery payload" />

Reviewer: ```suggestion
1. (Optional) Under **Alert Payload**, which allows you to customize how the alert notification will look, enter a JSON object in the format required by the target endpoint. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. <br/><img src={useBaseUrl('img/connection-and-integration/alert-payload.png')} alt="recovery payload" />
```

    :::note
    Variables are escaped according to the JSON standard, meaning that they can be used in application JSON. 
    :::
    You can test your customized alert payload by clicking **Test Alert Payload**, which will fire a sample webhook call to the given endpoint.

> 1. (Optional) Under **Recovery Payload**, you can customize your resolution notification payload JSON for an alert.<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload.png')} alt="recovery payload" /><br/>Example:<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload-example.png')} alt="recovery payload" /><br/>Clicking **Test Alert + Recovery Payload** will test your recovery alert payload by firing a sample webhook call to the given endpoint.

Reviewer: ```suggestion
1. (Optional) Under **Recovery Payload**, which allows you to customize how the recovery notification will look, enter a JSON object in the format required by the target webhook URL.<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload.png')} alt="recovery payload" /><br/>Example:<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload-example.png')} alt="recovery payload" /><br/>Clicking **Test Alert + Recovery Payload** will test your recovery alert payload by firing a sample webhook call to the given endpoint.
```

> 1. (Optional) Under **Recovery Payload**, you can customize your resolution notification payload JSON for an alert.<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload.png')} alt="recovery payload" /><br/>Example:<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload-example.png')} alt="recovery payload" /><br/>You can test your recovery alert payload by clicking **Test Alert + Recovery Payload**, which will fire a sample webhook call to the given endpoint.

Author: ```suggestion
1. (Optional) Under **Recovery Payload**, you can customize your resolution notification payload JSON for an alert.<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload.png')} alt="recovery payload" /><br/>Example:<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload-example.png')} alt="recovery payload" /><br/>Clicking **Test Alert + Recovery Payload** will test your recovery alert payload by firing a sample webhook call to the given endpoint.
```
